### DISCUSSION TITLE
Edit Mode Issues?

### DISCUSSION THREAD

**Discussion by shockerNY** [ggarra13/mrv2#255] [Q&A]
Version used v1.1.9
I have changed the default setting to start the application in Edit mode.

In Edit mode, the application crashes when I first press "Ctrl+X" to cut.

If I first perform "Ctrl+C">"Ctrl+V" for copy/paste and then "Ctrl+X", it works fine and the application does not crash.

So far, it is reproducible for sure. I thought it might be a bug and reported it.

**Comment by ggarra13** [comment]
> Version used v1.1.9 I have changed the default setting to start the application in Edit mode.
> 
> So far, it is reproducible for sure. I thought it might be a bug and reported it.

Please provide OS and more information like if it happens with a movie, with a sequence of images, with an EDL Playlist, with an .otio file, or with any of them, etc.

**Reply by shockerNY** [reply]
OK.
Additional information.

PC
Windows 10 Pro 22H2
OS Build: 19045.4598

Intel(R) Core(TM) i7-10700K CPU @ 3.80 GHz 3.80 GHz
RAM 64.0 GB
GPU RTX 2070 SUPER


Tried and tested.
mov Prores 422LT
mov DNxHR
png sequence

I am using this as a preview of the animation, so I have not tried anything else.


Other info.
It definitely fails in mrv2-v1.1.9.
I haven't tried it on all versions, but it worked fine on mrv2-v1.1.3.


That's all. Do you have enough information?

**Reply by ggarra13** [reply]
Thanks for the info, but I am afraid I cannot reproduce it.  I've tried it with several movies, EXR sequences, TGA sequences and it works fine for me.

Does it fail with this open source movie:

https://download.blender.org/demo/movies/BBB/bbb_sunflower_1080p_30fps_normal.mp4.zip
